src/utils.rs

```diff
@@ -70,54 +71,49 @@ pub fn _stringify_sets_using_map(
 }
 
 pub fn convert_list_of_tuples_to_hashmap(
-    list_of_tuples: &Vec<(TermID, PredicateSetKey, TermID)>,
```

Comment:
Are all these type aliases changed because of `let predicate_set_key: PredicateSetKey = predicate_set_to_key(predicates);` on line 82?

Reply:
I think this is a (harmless?) mistake - I'll change it back, these should be aliases